include/deal.II/base/exceptions.h

```diff
@@ -1086,6 +1086,15 @@ namespace StandardExceptions
     "if deal.II was configured to use LAPACK, but cmake did not "
     "find a valid LAPACK library.");
 
+  /**
+   * This function requires support for the MPI library.
+   */
+  DeclExceptionMsg(
+    ExcNeedsMPI,
+    "You are attempting to use functionality that is only available "
+    "if deal.II was configured to use MPI, but cmake did not "
+    "find a valid MPI library.");
```

Comment:
The user needs to explicitly enable DEAL_II_WITH_MPI, so the second part of this sentence doesn't make much sense to me.

Reply:
OK. I have remove that part of the sentence.